shell-completion/bash/systemctl.in

```diff
@@ -68,35 +68,35 @@ __filter_units_by_properties () {
 
 __get_all_units      () { { __systemctl $1 list-unit-files; __systemctl $1 list-units --all; } \
         | { while read -r a b; do [[ $a =~ @\. ]] || echo " $a"; done; }; }
-__get_template_names () { __systemctl $1 list-unit-files \
+__get_template_names () { __systemctl $1 list-unit-files "$2*" \
         | { while read -r a b; do [[ $a =~ @\. ]] && echo " ${a%%@.*}@"; done; }; }
 
-__get_active_units   () { __systemctl $1 list-units       \
+__get_active_units   () { __systemctl $1 list-units      "$2*" \
         | { while read -r a b; do echo " $a"; done; }; }
 __get_startable_units () {
         # find startable inactive units
         __filter_units_by_properties $1 ActiveState,CanStart inactive,yes $(
-            { __systemctl $1 list-unit-files --state enabled,enabled-runtime,linked,linked-runtime,static,indirect,disabled,generated,transient | \
+            { __systemctl $1 list-unit-files --state enabled,enabled-runtime,linked,linked-runtime,static,indirect,disabled,generated,transient "$2*" | \
                       { while read -r a b; do [[ $a =~ @\. ]] || echo " $a"; done; }
-              __systemctl $1 list-units --state inactive,failed | \
+              __systemctl $1 list-units --state inactive,failed "$2*" | \
                       { while read -r a b c; do [[ $b == "loaded" ]] && echo " $a"; done; }
             } | sort -u )
 }
 __get_restartable_units () {
         # filter out masked and not-found
         __filter_units_by_property $1 CanStart yes $(
-            __systemctl $1 list-unit-files --state enabled,disabled,static | \
+            __systemctl $1 list-unit-files --state enabled,disabled,static "$2*" | \
                     { while read -r a b; do [[ $a =~ @\. ]] || echo " $a"; done; }
-            __systemctl $1 list-units | \
+            __systemctl $1 list-units "$2*" | \
                     { while read -r a b; do echo " $a"; done; } )
 }
-__get_failed_units   () { __systemctl $1 list-units       \
+__get_failed_units   () { __systemctl $1 list-units      "$2*" \
```

Comment:
I do not like this 'indentation'...

Reply:
File already has funky indentation like this. Sure though - will move the new argument just after list-units.